frontend/packages/operator-lifecycle-manager/src/dev-catalog.ts

```diff
@@ -67,7 +67,7 @@ export const normalizeClusterServiceVersions = (
       href: `/ns/${desc.csv.metadata.namespace}/clusterserviceversions/${
         desc.csv.metadata.name
       }/${referenceForProvidedAPI(desc)}/~new`,
-      supportUrl: null,
+      supportUrl: _.get(desc, ['csv', 'metadata', 'annotations', 'marketplace.openshift.io/support-workflow'], ''),
```

Comment:
Prefer optional chaining

```suggestion
      supportUrl: desc.csv.metadata.annotations?.['marketplace.openshift.io/support-workflow'],
```

https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#optional-chaining